res/controllers/Traktor-Kontrol-Z1-scripts.js

```diff
@@ -0,0 +1,252 @@
+        message.addControl(group, name, offset, "H", bitmask);
+        message.setCallback(group, name, callback);
+    }
+
+    registerOutputPackets() {
+        const output = new HIDPacket("output", 0x80);
+
+        output.addOutput("[ControlX]", "mode", 0x13, "B");
+
+        output.addOutput("[Channel1]", "pfl", 0xF, "B");
+        output.addOutput("[Channel2]", "pfl", 0x10, "B");
+
+        output.addOutput("[Channel1]", "play_indicator", 0x11, "B");
+        output.addOutput("[Channel2]", "play_indicator", 0x14, "B");
+
+        output.addOutput("[QuickEffectRack1_[Channel1]]", "enabled", 0x12, "B");
+        output.addOutput("[QuickEffectRack1_[Channel2]]", "enabled", 0x15, "B");
+
+        output.addOutput("[Channel1]", "vu-30", 0x1, "B");
+        output.addOutput("[Channel1]", "vu-15", 0x2, "B");
+        output.addOutput("[Channel1]", "vu-6", 0x3, "B");
+        output.addOutput("[Channel1]", "vu-3", 0x4, "B");
+        output.addOutput("[Channel1]", "vu0", 0x5, "B");
+        output.addOutput("[Channel1]", "vu3", 0x6, "B");
+        output.addOutput("[Channel1]", "vu6", 0x7, "B");
+
+        output.addOutput("[Channel2]", "vu-30", 0x8, "B");
+        output.addOutput("[Channel2]", "vu-15", 0x9, "B");
+        output.addOutput("[Channel2]", "vu-6", 0xA, "B");
+        output.addOutput("[Channel2]", "vu-3", 0xB, "B");
+        output.addOutput("[Channel2]", "vu0", 0xC, "B");
+        output.addOutput("[Channel2]", "vu3", 0xD, "B");
+        output.addOutput("[Channel2]", "vu6", 0xE, "B");
+
+        this.controller.registerOutputPacket(output);
+
+        engine.makeConnection("[QuickEffectRack1_[Channel1]]", "enabled", this.outputHandler.bind(this));
+        engine.makeConnection("[QuickEffectRack1_[Channel2]]", "enabled", this.outputHandler.bind(this));
+
+        engine.makeConnection("[Channel1]", "pfl", this.outputHandler.bind(this));
+        engine.makeConnection("[Channel2]", "pfl", this.outputHandler.bind(this));
+
+        this.vuLeftConnection = engine.makeUnbufferedConnection("[Channel1]", "vu_meter", this.vuMeterHandler.bind(this));
+        this.vuRightConnection = engine.makeUnbufferedConnection("[Channel2]", "vu_meter", this.vuMeterHandler.bind(this));
+
+        this.lightDeck(false);
+    }
+
+    modeHandler(field) {
+        this.modePressed = field.value;
+        this.outputHandler(field.value, field.group, "mode");
+    }
+
+    headphoneHandler(field) {
+        if (field.value === 0) {
+            return;
+        }
+        // Go to cue and stop when modifier is active
+        if (this.modePressed) {
+            engine.setValue(field.group, "cue_gotoandstop", field.value);
+        } else {
+            script.toggleControl(field.group, "pfl");
+        }
+    }
+
+    fxHandler(field) {
+        if (field.value === 0) {
+            // Always clear play indicator on button release
+            this.controller.setOutput(field.group, "play_indicator", 0x0, true);
```

Comment:
```suggestion
            this.controller.setOutput(field.group, "play_indicator", 0x00, true);
```
Let's always use both digits for the hex-values with 7bits